cfme/containers/provider/openshift.py

```diff
@@ -63,6 +63,27 @@ def view_value_mapping(self):
         return out
 
 
+class PrometheusEndpoint(DefaultEndpoint):
+    """Represents prometheus Endpoint"""
+    name = 'prometheus'
+
+    @property
+    def view_value_mapping(self):
+        out = {
```

Comment:
I created a new class for Prometheus based endpoints.
Both Prometheus Alerts and Prometheus matrics classes to inherit for that base class, the base class contains a view_value_mapping function.

since both endpoints exist only from 5.9, I used version pick in the base class.

@mshriver What do you think? Is this solution is good enough?

Reply:
See the comment above[1]

[1] https://github.com/ManageIQ/integration_tests/pull/6707#discussion_r172028869